src/test/regress/sql/multi_extension.sql

```diff
@@ -24,60 +24,16 @@ WHERE pgd.refclassid = 'pg_extension'::regclass AND
 DROP EXTENSION citus;
 \c
 
--- Create extension in oldest version, test every upgrade step
+-- Create extension in oldest version, this is expected to fail
```

Comment:
Add these commands back. Create a hidden GUC (we have several, check the code) that is settable by SQL. Call it `citus.enable_version_checks` and have it default to `true`.

If it's `false`, all the `ErrorIf` functions you wrote should just exit early.

Set it to `false` here and add back these upgrades.